src/pages/UserProfilePage.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from 'material-ui/styles';
+import Drawer from 'material-ui/Drawer';
+import AppBar from 'material-ui/AppBar';
+import Toolbar from 'material-ui/Toolbar';
+import List from 'material-ui/List';
+import Typography from 'material-ui/Typography';
+import Divider from 'material-ui/Divider';
+import { eventsFolderListItems, profileFolderListItems } from '../assets/UserTileData';
+
+const drawerWidth = 240;
+
+const styles = theme => ({
+    root: {
+        flexGrow: 1,
```

Comment:
why do you have a flex-grow property and hard-bound height? Flex-grow makes the height flexible but the height declaration right after the flex-grow property nullifies it